lib/config-win32.h

```diff
@@ -709,7 +709,7 @@ Vista
 #endif
 
 /* Define to use the Windows crypto library. */
-#if !defined(USE_OPENSSL) && !defined(USE_NSS)
+#if defined(USE_SCHANNEL)
```

Comment:
You have changed it to use win32 crypto only if schannel but is that really what you meant? Aren't crypto and schannel two separate things, and now that there are changes in ntlm to handle win32 crypto properly can't we just enable USE_WIN32_CRYPTO all the time?

Reply:
Since we're in `config-win32.h`, `_WIN32` should always be defined, meaning that `USE_WIN32_CRYPTO` could just be enabled unconditionally in this header.

Reply:
It requires: `wincrypt.h`, `CryptAcquireContext()`, `CryptCreateHash()`, `CryptHashData()`, `CryptGetHashParam()`, `CryptDestroyHash()`, `CryptReleaseContext()`, `CryptImportKey()`, `CryptImportKey()`, `CryptDestroyKey()`. As per [MSDN](https://msdn.microsoft.com/library/windows/desktop/aa380252), these are available starting Windows XP/2003, so the condition required is `_WIN32_WINNT >= 0x0501`.

Reply:
@vszakats 
> As per MSDN, these are available starting Windows XP/2003

That's only because mention of old unsupported OSs are regularly removed from the MSDN documentation. https://msdn.microsoft.com/en-us/library/ms867086.aspx says it's at least available since Windows NT 4 and Windows 95.

Reply:
FYI Windows 2000 has all those functions xported in advapi32. Whether they're actually declared or not for Windows 2k I don't know (yes in original mingw), but we've never had an os version check on that symbol before and people do build for Win2k at least according to the survey. I think if you remove the guard it will be fine. If someone is building for Windows 98 (how?) we can wait for feedback if it breaks for them.

Reply:
Okay, I'll commit that and wait for a green flag of course. MSDN tends to be unreliable, exactly like @MarcelRaad says.